template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/AbstractRdsRoleConfigProvider.java

```diff
@@ -26,4 +27,7 @@ protected RdsView getRdsView(TemplatePreparationObject source) {
         return new RdsView(getRdsConfig(source));
     }
 
+    public static String dataBaseTypeForCM(DatabaseVendor vendor) {
```

Comment:
this should be presented only in you configprovider

Reply:
Ok. I thought it was some generic CM mechanism, but it looks like it's specific to out stuff:
```
      "validValues": [
        "mysql",
        "oracle",
        "postgresql"
      ],
```